frappe/core/doctype/communication/mixins.py

```diff
@@ -216,7 +216,11 @@ def get_assignees(self):
 			"reference_name": self.reference_name,
 			"reference_type": self.reference_doctype,
 		}
-		return ToDo.get_owners(filters)
+		
+		if self.reference_doctype == "ToDo" and self.reference_name != None:
```

Comment:
```suggestion
		if self.reference_doctype and self.reference_name:
```
Just make sure `reference_doctype` & `reference_name` have values.

All assignees should be notified about a new email on a document timeline.

---

Just for context we create a ToDo record to indicate an assignment. So if user "xyz" is assigned to "Ticket" "123" a ToDo is created for "xyz" in "Open" state with reference_doctype = "Ticket" & reference_name = "123"